From b1c83e8c1b92d1883717b1c276273d8ae365e31c Mon Sep 17 00:00:00 2001
From: Kexy Biscuit <kexybiscuit@aosc.io>
Date: Tue, 4 Feb 2025 14:14:35 +0800
Subject: [PATCH 297/301] AOSCOS: serial: 8250_pci: Move WCH CH384 4S1P card ID
 into pci_ids.h

Fix incomplete port of the previous patch mentioned below.

Fixes: 535a07698b8b ("serial: 8250_pci: Share WCH IDs with parport_serial driver")
Fixes: "AOSCOS: parport: Add support for the WCH384 4S/1P multi-IO card"
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/tty/serial/8250/8250_pci.c | 1 -
 include/linux/pci_ids.h            | 1 +
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/tty/serial/8250/8250_pci.c b/drivers/tty/serial/8250/8250_pci.c
index dd60ee8dd39d..9dcedb1ed30a 100644
--- a/drivers/tty/serial/8250/8250_pci.c
+++ b/drivers/tty/serial/8250/8250_pci.c
@@ -74,7 +74,6 @@
 #define PCI_DEVICE_ID_AMCC_ADDIDATA_APCI7800 0x818e
 
 #define PCI_DEVICE_ID_WCHIC_CH384_4S	0x3470
-#define PCI_DEVICE_ID_WCHIC_CH384_4S1P	0x3450
 #define PCI_DEVICE_ID_WCHIC_CH384_8S	0x3853
 
 #define PCI_DEVICE_ID_MOXA_CP102E	0x1024
diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index a66d3f64b6c5..7732a2ceb80b 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -2597,6 +2597,7 @@
 #define PCI_DEVICE_ID_WCHIC_CH382_0S1P	0x3050
 #define PCI_DEVICE_ID_WCHIC_CH382_2S1P	0x3250
 #define PCI_DEVICE_ID_WCHIC_CH382_2S	0x3253
+#define PCI_DEVICE_ID_WCHIC_CH384_4S1P	0x3450
 
 #define PCI_VENDOR_ID_SILICOM_DENMARK	0x1c2c
 
-- 
2.48.1

